src/custom/utils/switchToNetwork.ts

```diff
@@ -3,39 +3,20 @@ import { hexStripZeros } from '@ethersproject/bytes'
 import { ExternalProvider } from '@ethersproject/providers'
 import { CHAIN_INFO } from 'constants/chainInfo'
 import { SupportedChainId } from 'constants/chains'
-import { INFURA_NETWORK_URLS } from 'constants/infura'
+import { RPC_NETWORKS } from '../connectors'
 
 interface SwitchNetworkArguments {
   provider: ExternalProvider
   chainId: SupportedChainId
 }
 
 export function getRpcUrls(chainId: SupportedChainId): [string] {
-  switch (chainId) {
```

Comment:
I can kill of this code because `RPC_NETWORKS` has already **all the supported networks** rpc endpoints.

So it should be equivalent. I also verified we don't have a different INFURA_KEY in our secrets, and the value match the infura key matches the one in `INFURA_NETWORK_URLS`

Reply:
While I do agree this makes sense, I would feel more comfortable if we do this change independently, and not on a late Friday afternoon.

I suggest you stick to the minimal changes only and do the refactor to develop branch to be on the safe side